### ISSUE TITLE
Instructions how to create the media & various renditions in dash etc

### ISSUE THREAD

**Issue by kaihendry** [GoogleChrome/sample-media-pwa#16]
Would be great if you could please document how I can create my own bucket with my own transcoded videos, assuming say I'm starting from the FCPX output of some huge4k.mov file.

https://github.com/GoogleChrome/sample-media-pwa/blob/master/src/client/videos.json

**Comment by WORMSS** [comment]
Devil's Advocate here.. Would that not be a question to ask the people who made Dash ??

**Comment by kaihendry** [comment]
@WORMSS That would be like asking WHATWG how to build a sample-media-pwa... no? :rofl:

**Comment by WORMSS** [comment]
No.. It would be like asking the people who made Dash, "How to make media and various renditions in Dash"

**Comment by kaihendry** [comment]
Well, I would still prefer to hear how Paul did it.  I did catch a glimpse of how he did it with various ffmpeg invocations.

**Comment by paullewis** [comment]
I'd be happy to provide it. Though I believe @jpmedley is working on documenting all of that good stuff, so it might be better to wait so you get a proper and thorough explanation rather than my hacked-up version!

**Comment by j2is** [comment]
Any update on this?

**Comment by joeyparrish** [comment]
Please check out the recently-released Shaka Streamer:

https://github.com/google/shaka-streamer

You provide one config file for the input, one for the encoding pipeline, and optionally a cloud storage bucket (Google or Amazon) to push the output to.  It runs FFmpeg to transcode and Shaka Packager to package, optionally encrypt, and write the output manifests (DASH, HLS, or both).  No messy command lines, no need to be an FFmpeg expert.

There are many sample configs in the repo, and full docs are available here:

https://google.github.io/shaka-streamer/

Does this help?

**Comment by j2is** [comment]
Ah perfect, thank you!
